DSP_ SSP_Coordination.md

```diff
@@ -171,11 +172,38 @@ OpenRTB Core BidRequest.imp = {
 }
 ```
 
+Optionally we support also "ae" and "biddable" fields given in alternative sections of imp object. Like in imp -> ext -> igs
```

Comment:
let's add a link to IAB specs that we reference

Reply:
Done